rules/cloud/aws_route_53_domain_transferred_to_another_account.yml

```diff
@@ -0,0 +1,22 @@
+title: AWS Route 53 Domain Transferred to Another Account
```

Comment:
`id: b056de1a-6e6e-4e40-a67e-97c9808cf41b`
use https://www.uuidgenerator.net/version4 to get one